src/wabt.cpp

```diff
@@ -636,6 +636,11 @@ ExecutionResult WabtEngine::execute(
   );
 
   ensureCondition(Succeeded(loadResult) && module, ContractValidationFailure, "Module failed to load.");
+
+  for (auto it = errors.begin(); it != errors.end(); ++it) {
```

Comment:
Can you wrap these under `#if HERA_DEBUGGING`?

Also I don't think this will work at all, because the `ensureCondition(Succeeded(loadResult)` will abort execution before this for loop.

Reply:
Oh. true.